content/docs/user-guide/external-dependencies.md

```diff
@@ -146,27 +146,39 @@ $ dvc run -n download_file \
 
 </details>
 
-## Example: DVC remote aliases
+## Example: Using DVC remote aliases
 
-If instead of a URL you'd like to use an alias that can be managed
-independently, or if the external dependency location requires access
-credentials, you may use `dvc remote add` to define this location as a DVC
-Remote, and then use a special URL with format `remote://{remote_name}/{path}`
-to define an external dependency.
+You may want to encapsulate external locations as configurable entities that can
+be managed independently. This is useful if multiple dependencies (or stages)
+reuse the same location, or if its likely to change in the future. And if the
+location requires authentication, you need a way to configure it in order to
+connect.
 
-For example, for an HTTPs remote/dependency:
+[DVC remotes](/doc/command-reference/remote) can do just this. You may use
+`dvc remote add` to define them, and then use a special URL with format
+`remote://{remote_name}/{path}` (remote alias) to define the external
+dependency.
+
+Let's see an example using SSH. First, register and configure the remote:
+
+```dvc
+$ dvc remote add myssh ssh://myserver.com
+$ dvc remote modify --local myssh user myuser
+$ dvc remote modify --local myssh password mypassword
+```
+
+> Please refer to `dvc remote add` for more details like setting up access
+> credentials for the different remote types.
+
+Now, use an alias to this remote when defining the stage:
 
 ```dvc
```

Comment:
I also realized this doc only uses `dvc run` to define stages and doesn't show the resulting dvc.yaml files (except for import(-url) examples. Maybe we shouldn't even use `dvc run` in most of these examples? Since manually writing dvc.yaml is the recommended way.

But then again you can't easily write an external dependency in dvc.yaml ... Hmmm 🤔